src/index.js

```diff
@@ -113,7 +148,7 @@ function pullRequestParser(content) {
  * @returns {string} - parsed message.
  */
 function issuesParser(content) {
-  let parsedTask = `[${content.title.replace(/[[\]']+/g, '')}](${content.url})`;
+  let parsedTask = `${createTaskBadge(content.url)}: <a href="${content.url}">${escapeChars(content.title, '')}</a>`;
```

Comment:
Why `escapeChars` contain two arguments.

Reply:
oh. sorry. have not refactor. i'll fix